src/translations/en/common.json

```diff
@@ -82,6 +82,7 @@
   "migrationNote": "Be aware that settings are not migrated between different firmwares, make sure to take note of your motor directions and other settings you might want to move over. Settings will be migrated between different versions of the same firmware.",
   "escDirectionReversed": "Direction Reversed",
   "escBidirectionalMode": "Bidirectional Mode",
+  "escSelectProtocol": "Select ESC Protocol",
```

Comment:
Please also add a Hint int the hint translation file to explain in detail what this setting does.